spec/features/course/assessment/submission/programming_answer_comment_spec.rb

```diff
@@ -15,6 +15,18 @@
 
     context 'As a course student' do
       let(:user) { student }
+      let(:answers) { assessment.questions.attempt(submission) }
+      def annotation
```

Comment:
@lowjoel Why not just use a let block ?  I don't think defining method in specs is good..